modules/admin/src/views/Endpoints.vue

```diff
@@ -198,7 +203,7 @@ export default {
           ])
         }, []))
 
-        this.selectedColumns = ['_id']
+        this.selectedColumns = columns.slice(0, 5)
```

Comment:
Why only the first five columns? Can you add a comment explaining this?

Reply:
it is because we do not know the columns until after first request.. this page is a generic page for any endpoint, in future we want to use this as a base to configure endpoints and define columns there

Reply:
correct